.buildkite/pipeline.yaml

```diff
@@ -0,0 +1,24 @@
+steps:
+  - label: ":eslint: Lint"
+    command:
+      - "yarn install"
+      - "yarn lint"
+    plugins:
+      - docker#v3.0.1:
+          image: "node:10"
+
+  - label: ":karma: Tests"
+    command:
+      - "yarn install"
```

Comment:
Is it really worth it to have separate steps like this? (I am assuming steps run in parallel, but please correct me if that's wrong...) I would assume `yarn install` is the longest part of the process, so having to repeat it 3 times feels a bit wasteful...

Reply:
It sure isn't. That's what "better pipelines" is though: https://github.com/vector-im/riot-web/issues/9019

(rationale: fixing it now is not fun because we'll want to think carefully about what we actually want. The purpose of this exercise it to ditch Travis CI and migrate over to something useful, bringing whatever we have along with it).

Things are parallel.